tools/gulp/tasks/docs.ts

```diff
@@ -177,3 +180,197 @@ function createTagNameAliaser(classPrefix: string) {
     return this;
   };
 }
+
+interface ExampleMetadata {
```

Comment:
Ok, I started that at first but moved into the same file at the end cuz of similarities .